pkg/modulereader/resreader.go

```diff
@@ -53,62 +53,50 @@ type OutputInfo struct {
 // strings or as a list of maps directly into OutputInfo struct
 func (mo *OutputInfo) UnmarshalYAML(value *yaml.Node) error {
 	var name string
+	const yamlErrorMsg string = "block beginning at line %d: %s"
```

Comment:
nit. It seems like you always use the same value of `value.Line`, let's make it a part of this const.